src/item.cpp

```diff
@@ -6597,10 +6600,6 @@ int item::price_no_contents( bool practical ) const
         price -= price * static_cast< double >( damage_level() ) / 5;
     }
 
-    if( is_filthy() ) {
-        price *= 0.8;
```

Comment:
You can keep both, but with what sensible prices are for most items, it probably doesn't make a difference.

Reply:
We can keep both, but I feel having only the fixed malus better reflects that a valuable item is still very valuable while filthy.

It also makes it easier to calculate prices for the test cases I didn't write. :)